pages/glossary/background-colors-of-element.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Background Colors Of Element
+key: background-colors-of-element
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The colors of all pixels, excluding the [foreground colors](#foreground-colors-of-text) pixels of the element's [text nodes][], in the element's [bounding client rectangle](https://drafts.csswg.org/cssom-view/#dom-element-getboundingclientrect).
```

Comment:
Why not just say that all the pixels that change when the background property is set to transparent is the background color? The way this is done now results in just about anything becoming part of the background. Jut about any content element that isn't a text node is now treated as background, including img, input, iframe, etc. Also, text of other elements can even be considered part of the background with this definition.

Reply:
That was my goal, because that is what's presented on the page. Do you find this problematic?